modules/search/class.jetpack-search-widget-filters.php

```diff
@@ -96,7 +96,7 @@ public function enqueue_frontend_scripts() {
 
 	private function get_sort_types() {
 		return array(
-			'relevance|DESC' => esc_html__( 'Relevance', 'jetpack' ),
+			'relevance|DESC' => esc_html__( 'Relevance (recommended)', 'jetpack' ),
```

Comment:
This is showing up on the front end, but should only be shown in the admin.

Reply:
Fixed - thanks!